tests/FakeItEasy.Tests/Builders/BuilderForParsedArgumentExpression.cs

```diff
@@ -14,7 +14,7 @@ internal class BuilderForParsedArgumentExpression : TestDataBuilder<ParsedArgume
 
         public BuilderForParsedArgumentExpression()
         {
-            this.FromFirstArgumentInMethodCall(() => this.Equals(null));
+            this.FromFirstArgumentInMethodCall(() => object.ReferenceEquals(this, null));

```

Comment:
Doesn't it change the behavior? Apparently, `FromFirstArgument` extracts the first parameter from the expression, which was `null`, but is now `this`. I'm not sure what this class is doing; maybe it doesn't matter.